dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/JobConfigurationRunErrorsControllerTest.java

```diff
@@ -174,7 +174,7 @@ void testGetJobRunErrors_ListIncludeInput() throws InterruptedException {
     // language=JSON
     String expected =
         """
-      {"trackedEntities":[{"trackedEntity":"sHH8mh1Fn0z","trackedEntityType":{"idScheme":"UID","identifier":"nEenWmSyUEp"},"orgUnit":{"idScheme":"UID","identifier":"DiszpKrYNg7"},"inactive":false,"deleted":false,"potentialDuplicate":false,"relationships":[],"attributes":[],"enrollments":[]}],"enrollments":[],"events":[],"relationships":[]}""";
```

Comment:
Does this change affect jobs? If so should we add a migration to remove unnecessary data? Or would such jobs be removed after a while anyway

Reply:
I never know :grimacing: 
@jbee Can you help us here?

Reply:
Tracker import jobs are removed a day after they have been running. I am not quite sure what changed with regards to the job but I think you removed something which should be fine. If the job was stored with more parameter data than it later executes this still works.

Reply:
Thanks @jbee .